corpora/csip/metadata/metshdr/CSIP7/testCase.xml

```diff
@@ -1,23 +1,51 @@
-<!-- Root element for an individual test case, allows these to be wrapped into
-XML lists -->
-<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
-  <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP7"/>
-  <!-- URL references to requirements for convenient lookup -->
+<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd" testable="TRUE">
+  <id specification="CSIP" version="2.0.4" requirementId="CSIP7"/>
   <references>
     <reference requirementId="CSIP7" URL="http://earkcsip.dilcis.eu/#CSIP7"/>
   </references>
-  <!-- The full text of the requirement. -->
-  <requirementText>@CREATEDATE describes the date of creation of the package.
-</requirementText>
-  <!-- Textual description of the test case, extra notes beyond the requirment text. -->
-  <description></description>
-  <!-- List of requirments that this test case depends on in addition to the
-  main requirememt, e.g. general requirments on the form of IDs -->
-  <dependencies>
-  </dependencies>
-  <!-- A list of the validation rules derived from the test case -->
+  <requirementText>
+    <name>Package creation date</name>
+    <location>mets/metsHdr/@CREATEDATE</location>
+    <description>
+      Records the date the package was created.
+    </description>
+    <cardinality>1..1</cardinality>
+    <level>MUST</level>
+  </requirementText>
+  <description>
+    mets/metsHdr/@CREATEDATE MUST exist.
+    mets/metsHdr/@CREATEDATE MUST have a value that is a date.
+    mets/metsHdr/@CREATEDATE value MUST be the actual date the package was created.
+  </description>
+  <dependencies />
   <rules>
+    <rule id="1">
+      <description>mets/metsHdr/@CREATEDATE MUST exist.</description>
+      <error level="ERROR">
+        <message>
+          mets/metsHdr/@CREATEDATE attribute does not exist. MUST be: attribute exists and has a value that is a date in the format of xsd:dateTime.
+        </message>
+      </error>
+      <corpusPackages>
+        <package name="mets-xml_metsHdr_CREATEDATE_not_exist" isValid="FALSE">
+          <path></path>
+          <description>
+            METS.xml file has no mets/metsHdr/@CREATEDATE attribute.
+          </description>
+        </package>
+        <package name="minimal_IP_with_1_representation" isValid="TRUE">
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements.
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <!--    Requirement "mets/metsHdr/@CREATEDATE MUST have a value that is a 
+            date" is tested for using XML Schema validation as mets.xsd defines
+            the attribute as type="xsd:dateTime". -->
+      
+    <!--    Requirement "mets/metsHdr/@CREATEDATE value MUST be the actual date
```

Comment:
I think it would be possible and actually make sense to test if the date is after today.

Reply:
Thanks, good point! Fixed now.

Reply:
Beware here be dragons, time zones or even server clock discrepancies, and other oddities might make this an annoying test IRL. Perhaps with a suitable buffer-zone?

Reply:
Agreed, less than a day before the cliff is way too close. I changed it to 2037-01-01.